src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/createaccessrequirement/CreateManagedACTAccessRequirementStep2ViewImpl.java

```diff
@@ -46,11 +48,11 @@
 	@Inject
 	public CreateManagedACTAccessRequirementStep2ViewImpl(Binder binder) {
 		widget = binder.createAndBindUi(this);
-		editWikiButton.addClickHandler(new ClickHandler() {
-			@Override
-			public void onClick(ClickEvent event) {
-				presenter.onEditWiki();
-			}
+		editWikiButton.addClickHandler(event -> {
+			presenter.onEditWiki();
+		});
+		iduCheckbox.addValueChangeHandler(event -> {
+			intendedDataUsePublicCheckbox.setEnabled(event.getValue());
 		});
```

Comment:
When the IDU required checkbox is clicked, update enable state of the "is IDU public" checkbox (not applicable if IDU is optional).  Is also updated below, when it's initialized from an existing AR.